### ISSUE TITLE
Make Search feature more accessible

### ISSUE THREAD

**Issue by tomsedory** [OpenEugene/little-help-book-web#183]
Make following changes to all html files so Search feature in header is more accessible:

- Fix accessibility error "Image button missing alternative text" by adding alt="" to Search images. #175 
- Fix accessibility error "Missing form label" by adding aria-label="" to search images, search button and search field. #176
- Fix accessibility error "Orphaned form label" by adding id="search" to search button to tie label to button.

- [x] about.html (closed via PR #179)
- [x] category.html
- [x] emergency.html
- [x] index.html
- [x] map.html
- [x] provider.html
- [x] providers.html
- [x] subcategory.html
- [x] terms-of-use.html
